task2/src/components/AppTitle/AppTitle.jsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const AppTitleStyled = styled.span`
+    position: relative;
```

Comment:
what is it for?)

Reply:
It's interesting question. It's because of filter: blur prop on header part. I'll tell it more details at the meeting.